pallets/parachain-staking/src/lib.rs

```diff
@@ -0,0 +1,1754 @@
+// Copyright 2019-2022 PureStake Inc.
```

Comment:
Not sure we should keep header. We forked tx-pause and collator-selection, you can take a look at both pallets' header.